src/main/python/deep_qa/layers/attention/attention.py

```diff
@@ -49,6 +49,7 @@ def call(self, inputs, mask=None):
         vector, matrix = inputs
         matrix_mask = mask[1]
         num_rows = K.int_shape(matrix)[1]
+        print("Matrix shape:", K.int_shape(matrix))
```

Comment:
nevermind, i fixed it in https://github.com/allenai/deep_qa/pull/153/commits/e6666a2f5d44e96c9ddd8fe51020584818081098